### PR TITLE
doc: show create schema

### PR DESCRIPTION
**Pull request by xiongjiwei**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs/issues/9589

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-create-database.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: SHOW CREATE DATABASE
+summary: TiDB 数据库中 SHOW CREATE DATABASE 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-create-database/','/docs-cn/dev/reference/sql/statements/show-create-database/']
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-create-database.md:4]:
```suggestion
```

**Comment by TomShawn** [comment]
@xiongjiwei 能在 TOC.md 中添加入口吗？

File: sql-statements/sql-statement-show-create-database.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+---
+title: SHOW CREATE DATABASE
+summary: TiDB 数据库中 SHOW CREATE DATABASE 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-create-database/','/docs-cn/dev/reference/sql/statements/show-create-database/']
+---
+
+# SHOW CREATE TABLE
+
+`SHOW CREATE TABLE` 语句用于显示用 SQL 重新创建已有库的确切语句。
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-create-database.md:9]: 是不是查看创建数据库语句 CREATE TABLE 的详细信息呢？

Comment (qiancai) [reply sql-statements/sql-statement-show-create-database.md:9]: 看起来是针对一个已经创建的 table，查看此 table 当时 create 时用的 SQL 语句是什么

File: sql-statements/sql-statement-show-create-database.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
+---
+title: SHOW CREATE DATABASE
+summary: TiDB 数据库中 SHOW CREATE DATABASE 的使用概况。
+aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-create-database/','/docs-cn/dev/reference/sql/statements/show-create-database/']
+---
+
+# SHOW CREATE TABLE
```
Comment (TomShawn) [inline sql-statements/sql-statement-show-create-database.md:7]:
```suggestion
# SHOW CREATE DATABASE

```

**Review by tangenta — APPROVED** [review]

**Comment by xiongjiwei** [comment]
> @xiongjiwei 能在 TOC.md 中添加入口吗？

done

File: sql-statements/sql-statement-show-create-database.md (hunk 2 above)
Comment (xiongjiwei) [reply sql-statements/sql-statement-show-create-database.md]: sql_quote_show_create 没有作用

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**